core/CronArchive.php

```diff
@@ -1457,7 +1457,7 @@ private function makeWebsiteIdArchiveList(array $websitesIds)
             return new FixedSiteIds($websitesIds);
         }
 
-        if (!empty($this->shouldArchiveSpecifiedSites)) {
+        if (!empty($this->shouldArchiveSpecifiedSites) && !SharedSiteIds::isSupported()) {
```

Comment:
From a logical point of view, shouldn't that be an OR condition?

Reply:
Yes it should, fixed! :+1: